### DISCUSSION TITLE
where to start programming with java spring boot

### DISCUSSION THREAD

**Discussion by lutfiadysh** [community/community#61440] [Programming Help] [answered]
### Body

hello i want to start learing with java spring boott

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by RahmadFani** [comment, marked as answer]
Hello there

**Comment by MateoRodriguez0** [comment]
Hola soy de colombia y he aprendido a traves de cursos, puedes tomar cursos en udemy son muy buenos. Tambien aprende a usar la documentacion oficial de spring boot y lo mas importante en estos tiempos sacale provecho a la IA, me ha ayudado mucho en mis proyectos
